models/articles.model.js

```diff
@@ -1,42 +1,46 @@
 const db = require("../db/connection");
 
-exports.insertComment = (article_id, username, body) => {
-  if (!username || !body) {
-    return Promise.reject({ status: 400, msg: "400: Bad Request - Missing required fields" });
+exports.updateArticleVotes = (article_id, inc_votes) => {
+  if (inc_votes === undefined || typeof inc_votes !== 'number') {
```

Comment:
you don't need to check for the type of votes here because if its not a number or undefined sql will throw and error for you as you would be trying to add votes to undefined or a different type in your sql string